docs-preview/package.json

```diff
@@ -0,0 +1,152 @@
+{
+  "name": "markdocs",
```

Comment:
Will update the naming and publisher related information in `package.json` and `README.md` in next PR to avoid too many changes to review.

Reply:
Hi @qinezh , looks good.